### PR TITLE
Reworked instantiation of BubbledChangeEventArgs.

### PR DESCRIPTION
**Pull request by mlessmann**
BubbledChangeEvents are now fired on delete as well.

### CODE REVIEW

**Comment by mlessmann** [comment]
Open question: BubbledChange is not being forwarded, because a deleted element has not parent anymore.
Possible solution: INotify***Chang**ing**

File: Models/Models/BubbledChangeEventArgs.cs [hunk 1]
```diff
@@ -73,27 +30,104 @@ public BubbledChangeEventArgs(IModelElement newElement)
         public EventArgs OriginalEventArgs { get; private set; }
 
         /// <summary>
-        /// Gets a value indicating whether the underlying change has been a elementary collection change
+        /// Gets the type of change that occured. This defines the type of OriginalEventArgs
+        /// and whether PropertyName is used.
         /// </summary>
-        public bool IsCollectionChangeEvent
```
Comment (georghinkel) [inline Models/Models/BubbledChangeEventArgs.cs]: Removing these properties will introduce a breaking change. Thus, they should remain, though they can be implemented using the change type information.

**Comment by georghinkel** [comment]
I guess this could be handled by propagating not the Deleted event itself, but capture this case in the implementation of setting a new parent, which for a deleted element means setting it to null.

**Comment by mlessmann** [comment]
Fixed accordingly.

**Pull request merged**